### PR TITLE
Add com.expidusos.calculator

### PR DESCRIPTION
**Pull request by RossComputerGuy**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: com.expidusos.calculator.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+app-id: com.expidusos.calculator
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
```
Comment (hfiguiere) [inline com.expidusos.calculator.yml:3]:
```suggestion
runtime-version: '23.08'
```


it's out now

File: flathub.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: this is disappointing.

Comment (hfiguiere) [reply flathub.json:2]: but I guess Flutter.....

Comment (RossComputerGuy) [reply flathub.json:2]: Yeah, aarch64 cross compiling for Flutter on x86_64 isn't really there yet from my experience. I want to support aarch64 but have had issues with trying to do so.

File: com.expidusos.calculator.yml (hunk 1 above)
Comment (RossComputerGuy) [reply com.expidusos.calculator.yml]: Alright, updated it.

**Comment by RossComputerGuy** [comment]
Fixed, had to manually update the artifacts since the CI release action screwed up and uploaded a tarball of a tarball of the build.

**Comment by hfiguiere** [comment]
bot, build com.expidusos.calculator

File: com.expidusos.calculator.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  - --device=dri
+  - --share=network
+modules:
+  - name: FileManager
+    buildsystem: simple
+    build-commands:
+      - cp -R bundle /app/expidus_calculator
+      - mkdir -p /app/bin && ln -s /app/expidus_calculator/calculator /app/bin/calculator
+      - chmod +x /app/bin/calculator
+      - mkdir -p /app/expidus_calculator/lib && mv /app/expidus_calculator/lib/*.so /app/expidus_calculator/lib
```
Comment (hfiguiere) [inline com.expidusos.calculator.yml:20]: why do you do that?
source and destination are the same.

Comment (RossComputerGuy) [reply com.expidusos.calculator.yml:20]: The manifest was copied from the file manager app and in https://github.com/flathub/com.expidusos.file_manager/pull/3, there was a weird issue where this had to be done. The issue is not present with the calculator for some reason despite using the same exact workflow steps. I just haven't removed the unnecessary move.

Comment (RossComputerGuy) [reply com.expidusos.calculator.yml]: Fixed

**Comment by hfiguiere** [comment]
bot, build com.expidusos.calculator

**Comment by RossComputerGuy** [comment]
bot, build com.expidusos.calculator

**Comment by RossComputerGuy** [comment]
Sweet, works.

**Pull request closed without merging**